index.html

```diff
@@ -0,0 +1,145 @@
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <button>Back this Project</button><br>
+      <button>Remind me</button>
+      <p>
+        <sub>All or nothing. This project will only be funded if it reaches its goal by Thu, November 26 2020 1:35 PM
+          CST.</sub>
+      </p>
+    </aside>
+    <nav>
+      <a href="#">Campaign</a> | <a href="#">FAQ</a> | <a href="#">Updates<sup>2</sup></a> | <a
+        href="#">Comments<sup>19</sup></a> | <a href="#">Community</a>
+    </nav>
+    <aside>
+      <nav>
+        <a href="#">STORY</a>
+        <a href="#">RISKS</a>
+        <a href="#">ENVIRONMENTAL
+          COMMITMENTS</a>
+      </nav>
+    </aside>
+    <article>
+      <h2>Story</h2>
+      <p><strong>Hi there! We're IOTA Inc.</strong> </p>
+
+      <p>We’ve re-designed the calendar to simplify your life. After two previously successful calendar Kickstarters,
+        we’re
+        back
+        with our 2021 version - now made to be as forgiving as a whiteboard, yet as beautiful as an art piece.</p>
+
+      <p>To preorder and make it a reality, click Back This Project! <a href="#">Press Kit</a></p>
+      <img src="articlegif.gif" alt="The calendar showcased on a desk">
+      <img src="complicated.png"
+        alt="When a complicated life met a simple solution, The IOTA Inc, Wall Calendar was born.">
+      <img src="minimal.png" alt="The calendar displayed with it's packaging.">
+      <img src="forgiving.gif" alt="Shows the calendar being marked up and then wiped away.">
+      <img src="beautiful.png" alt="The calendar with a beautiful plant on a desk">
+      <h2>Use it However You'd Like</h2>
+      <p>Plan vacations, birthdays, or anniversaries. Make a colored Gantt chart. Start bullet journaling. Count the
+        days as they
+        pass. This calendar provides just enough structure without getting in the way.</p>
+      <p>Funny thing is, it looks so nice, that you might just want to leave it alone.
+      </p>
+      <img src="markingup.gif" alt="Shows the calendar being marked up">
+      <h2>Testimonials from 2019 and 2020</h2>
+      <ul>
+        <li><em>"I'm a writer working on my first novel and the Minimalist calendar has been PERFECT for me. I love that
+            it reflects the
+            fluidity and non-stop reality of time and shows me in one view the entire year, past and present."
+          </em>–Laura, Portland
+          Oregon</li>
+        <li><em>“We got a lot of use as a family calendar keeping us on the same page… Now I’m hooked!”</em>–Jesse,
+          Denver Colorado</li>
+        <li><em>"Your calendar has had a pretty significant effect on my life. By forcing myself to realize that time is
+            passing, and
+            questioning how important the thing that I'm spending my hours on, I think I've truly been living a more
+            fulfilling
+            life"</em>–Phil, New York City
+        </li>
+      </ul>
```

Comment:
It feels like a lot of these list items are emphasized and it's unclear what sets them apart from the rest of the text around them beyond the fact that they're on a list. 

Additionally, you're quoting someone, and we have a few elements designed to cite quotes inline, for example, the `q` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q

Not a huge deal, but just wanted to comment on it so you consider meaning a bit more in the future.